next.config.js

```diff
@@ -42,161 +43,152 @@ const nextauthUrl = process.env.NEXTAUTH_URL
-      // !! WARN !!
-      ignoreBuildErrors: true,
-    },
-    async redirects() {
-      return [
-        {
-          source: '/account-activate/:slug*',
-          destination: '/open-app',
-          permanent: true,
-        },
-        {
-          source: '/competition/:slug*',
-          destination: '/open-app',
-          permanent: true,
-        },
-        {
-          source: '/donate-tree',
-          destination: '/open-app',
-          permanent: true,
-        },
-        {
-          source: '/donate-trees',
-          destination: '/open-app',
-          permanent: true,
-        },
-        {
-          source: '/donate-trees/:slug*',
-          destination: '/open-app',
-          permanent: true,
-        },
-        {
-          source: '/my-trees',
-          destination: '/login',
-          permanent: true,
-        },
-        {
-          source: '/project/:slug*',
-          destination: '/open-app',
-          permanent: true,
-        },
-        {
-          source: '/redeem',
-          destination: '/login',
-          permanent: true,
-        },
-        {
-          source: '/reset-password/:slug*',
-          destination: '/open-app',
-          permanent: true,
-        },
-        {
-          source: '/signup',
-          destination: '/open-app',
-          permanent: true,
-        },
-        {
-          source: '/signup/:slug*',
-          destination: '/open-app',
-          permanent: true,
-        },
-      ];
-    },
-    assetPrefix: hasAssetPrefix ? `${scheme}://${process.env.ASSET_PREFIX}` : '',
-    // Asset Prefix allows to use CDN for the generated js files
-    // https://nextjs.org/docs/api-reference/next.config.js/cdn-support-with-asset-prefix
+    // When all the Sentry configuration env variables are available/configured
+    // The Sentry webpack plugin gets pushed to the webpack plugins to build
+    // and upload the source maps to sentry.
+    // This is an alternative to manually uploading the source maps
+    // Note: This is disabled in development mode.
+    if (
+      SENTRY_DSN &&
+      SENTRY_ORG &&
+      SENTRY_PROJECT &&
+      SENTRY_AUTH_TOKEN &&
+      COMMIT_SHA &&
+      NODE_ENV === 'production'
+    ) {
+      config.plugins.push(
+        new SentryWebpackPlugin({
+          include: '.next',
+          ignore: ['node_modules'],
+          stripPrefix: ['webpack://_N_E/'],
+          urlPrefix: `~${basePath}/_next`,
+          release: COMMIT_SHA,
+        }),
+        new SentryWebpackPlugin({
+          include: '.next',
+          ignore: ['node_modules'],
+          stripPrefix: ['webpack://_N_E/'],
+          urlPrefix: `~${basePath}/_next`,
+          release: COMMIT_SHA,
+        })
+      );
+    }
+    return config;
+  },
+  basePath,
+  // your config for other plugins or the general next.js here...
+  devIndicators: {
+    autoPrerender: false,
+  },
+  env: {
+    MAPBOXGL_ACCESS_TOKEN: process.env.MAPBOXGL_ACCESS_TOKEN,
```

Comment:
do we still use Mapbox?

Reply:
yes in one place, will remove in this branch